src/main/java/com/github/onsdigital/babbage/template/handlebars/helpers/SectionsHelper.java

```diff
@@ -0,0 +1,53 @@
+package com.github.onsdigital.babbage.template.handlebars.helpers;
+
+import com.github.jknack.handlebars.Handlebars;
+import com.github.jknack.handlebars.Options;
+import com.github.onsdigital.babbage.template.handlebars.helpers.base.BabbageHandlebarsHelper;
+import com.github.onsdigital.babbage.template.handlebars.helpers.markdown.CustomMarkdownHelper;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by angela on 04/06/21.
+ */
+public enum SectionsHelper implements BabbageHandlebarsHelper<Object> {
+
+    totalWordCount {
+        @Override
+        public CharSequence apply(Object context, Options options) throws IOException {
+            if (options.isFalsy(context) || ((List<Object>) context).isEmpty()) {
```

Comment:
This may be a personal preference, but I quite like extracting private methods from conditionals like this as they return a boolean, so then we could have `if (optionsAreEmpty)` or something like that. I'm fine with leaving as is if you would prefer though

Reply:
add private method to check for isEmpty